Package.swift

```diff
@@ -0,0 +1,63 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let swiftSettings: [SwiftSetting] = [
+    .define("SPM"),
```

Comment:
I don't think you need this. There is an existing built-in flag `SWIFT_PACKAGE`.

Reply:
https://github.com/swiftlang/swift-package-manager/blob/main/Documentation/Usage.md#packaging-legacy-code

Nice!

Reply:
Thanks @crazytonyli !

Addressed via cce8320caae791836c1888d60fb61003b20610bb